### DISCUSSION TITLE
How to cite Ferrite in a scientific paper

### DISCUSSION THREAD

**Discussion by Drachta** [Ferrite-FEM/Ferrite.jl#449] [General]
Hello,

I use Ferrite for my research and I am currently writing my first article.
How should I cite Ferrite ? Is there a specific paper I should cite, or is the github url enough for you ?

Thanks for your hard work!

**Comment by fredrikekre** [comment]
In the "About" section to the right here: https://github.com/Ferrite-FEM/Ferrite.jl there is a "Cite this repository" option, does that work for you?

**Comment by Drachta** [comment]
Oh, I never noticed that before. Thank you!
